doc/toolbox-list.1.md

```diff
@@ -4,13 +4,33 @@
 toolbox\-list - List existing toolbox containers and images
 
 ## SYNOPSIS
-**toolbox list** [*--containers* | *-c*] [*--images* | *-i*]
+**toolbox list** [*OPTIONS*]
 
 ## DESCRIPTION
 
 Lists existing toolbox containers and images. These are OCI containers and
 images, which can be managed directly with tools like `buildah` and `podman`.
 
+Listed information about images:
```

Comment:
Let's wait a bit before we go too much into this part, because it will be influenced by https://github.com/containers/toolbox/issues/579